03.asynchronous/callback.js

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env node
+
+import sqlite3 from "sqlite3";
+import timers from "timers/promises";
+
+const db = new sqlite3.Database(":memory:");
+
+db.run(
+  "CREATE TABLE books(id INTEGER PRIMARY KEY, title TEXT NOT NULL UNIQUE)",
+  () => {
+    db.run("INSERT INTO books (title) VALUES (?)", ["book1"], () => {
```

Comment:
CREATE TABLE とスタイルを揃えるなら以下のようにしたほうがよいです (どちらに揃えてもよいですが) 。

```suggestion
    db.run("INSERT INTO books(title) VALUES (?)", ["book1"], () => {
```

Reply:
全てのSQL文のスタイルを統一しました。